File: sorting/sortMarkToys.js
const maximumToys = (prices, k) => {
  let count = 0;
  let sum = 0;
  prices.sort((a, b) => a - b);
  for (let i = 0; i < prices.length; i++) {
    sum += prices[i];
    while (sum <= k) {
      count++;
      break;
    }
  }
  return count;
};

console.log(maximumToys([1, 12, 5, 111, 200, 1000, 10], 50));

File: sorting/sortMarkToys.py
def maximumToys(prices, k):
    sorted_arr = sorted(prices)
    count = 0
    for i in sorted_arr:
        if (i < k):
            count += 1
            k -= i

    return count


print(maximumToys([3, 7, 2, 9, 4], 15))
